crates/ruff_python_semantic/src/definition.rs

```diff
@@ -33,14 +33,15 @@ pub enum ModuleKind {
 }
 
 /// A Python module.
-#[derive(Debug)]
+#[derive(Debug, Copy, Clone)]
 pub struct Module<'a> {
     pub kind: ModuleKind,
     pub source: ModuleSource<'a>,
     pub python_ast: &'a [Stmt],
 }
 
 impl<'a> Module<'a> {
+    /// Return the fully-qualified path of the module.
     pub fn path(&self) -> Option<&'a [String]> {
```

Comment:
Nit: I wonder if we should name this `qualified_name` instead. `Path` is ambigious because it could either be a file path or a module path.

Reply:
Good call.